appveyor.yml

```diff
@@ -0,0 +1,29 @@
+build: false
+
+environment:
+    PYTHON: "C:\\Python27"
+    PYTHON_VERSION: "2.7.8"
+    PYTHON_ARCH: "32"
+
+    CONAN_REFERENCE: "Libgeohash/2469d3c"
+    CONAN_USERNAME: "ebclark2"
+    CONAN_CHANNEL: "testing"
```

Comment:
Probably you want to add: ``- CONAN_STABLE_BRANCH_PATTERN: "stable/*"`` otherwise master branches are published to ``stable`` instead of ``testing``

Reply:
Same in travis.yml

Reply:
fixed